modules/openy_features/openy_node/modules/openy_node_category/config/install/field.field.node.program_subcategory.field_category_description.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
     - field.storage.node.field_category_description
     - node.type.program_subcategory
   module:
+    - datalayer
```

Comment:
Do we really need it here?

Reply:
We have datalayer enabled in this field, so we need this depenency